.circleci/config.yml

```diff
@@ -367,18 +318,18 @@ workflows:
             - build-align
       - build-make-docker:
           requires:
-            - start
-      - build-integration1:
-          requires:
-            - start
-      - build-integration2:
-          requires:
-            - start
-      - finish:
+            - build-align
+      - test-integration:
+          name: "test-integration-<<matrix.pdk>>(<<matrix.design>>)"
           requires:
             - build-align
-            - build-cktgen
-            - build-PlaceRouteHierFlow
-            - build-integration1
-            - build-integration2
+          matrix:
+            parameters:
+              pdk:
+                - "Finfet"
```

Comment:
We are currently only running PDKS containing the word "Finfet". Do we also want to run bulk somewhere?

Reply:
Thanks Parijat. 
We can Run these designs on bulk also. I checked BUlk and they were also clean (except 2 designs for which there was no GDS but the check passed)

Reply:
Or I can work with Arvind to clean the bulk. Then merge it.

Reply:
So Finfet is not really clean. We have multiple tests asserting DRC violations. The errors were not being propagated to the circleci status previously because of how the test was being called. 

Do you want to try and fix these violations or should we mask these errors again?

Reply:
I think we should let them run and put a mask on DRC more than 10 for each design and we can target to bring them to zero.
user_const design can be removed as it was a dummy design to check the user constraints. I am still working on this.

Reply:
> I think we should let them run and put a mask on DRC more than 10 for each design and we can target to bring them to zero.

Done

> user_const design can be removed as it was a dummy design to check the user constraints. I am still working on this.

I don't see a test for user_const in .circleci/config.yml. I may have accidentally removed the test

Reply:
@kkunal1408 you are two failures away from having a clean turnin now so I would suggest merging ASAP. I would suggest having @soneryaldiz review the main thrust of this PR (user_const).

Note that I made a --maxerrors flag in the pytest --runnightly call itself so you can adjust it yourself over time. Its not really clean until that number hits zero (the number is currently 10).